acra/configuring-maintaining/general-configuration/acra-authmanager.md

```diff
@@ -0,0 +1,131 @@
+---
+title: acra-authmanager
+bookCollapseSection: true
+---
+
+# acra-authmanager
+
+`acra-authmanager` is CLI utility for [acra-webconfig](/acra/configuring-maintaining/general-configuration/acra-webconfig.md) user management.
```

Comment:
What about using another form? If it makes actual difference of course.
```suggestion
`acra-authmanager` is CLI utility for [acra-webconfig]({{< ref "/acra/configuring-maintaining/general-configuration/acra-webconfig.md" >}}) user management.
```

Reply:
thanks, I just missed. copied from IDE and forgot to wrap with tag